src/csharp/Microsoft.Spark.Worker/Command/SqlCommandExecutor.cs

```diff
@@ -524,4 +528,168 @@ public IArrowArray[] Run(ReadOnlyMemory<IArrowArray> input)
             }
         }
     }
+
+    internal class ArrowGroupedMapCommandExecutor : ArrowSqlCommandExecutor
```

Comment:
Since `s_writeOutputStream` will be going away once we move to Arrow `0.14`, I will change the `s_writeOutputStream` field to be `internal`, and just derive from `SqlCommandExecutor`. That way when it gets removed, we won't be inheriting from `ArrowSqlCommandExecutor`.